### PR TITLE
Stable: Update to v4.2

### PR DESCRIPTION
**Pull request by realmazharhussain**

### CODE REVIEW

File: io.github.realmazharhussain.GdmSettings.yaml [hunk 1]
```diff
@@ -10,7 +10,6 @@ finish-args:
   - --socket=wayland
   - --socket=fallback-x11
   - --talk-name=org.freedesktop.Flatpak
-  - --filesystem=xdg-config/monitors.xml:ro
```
Comment (bbhtt) [inline io.github.realmazharhussain.GdmSettings.yaml:13]: Hi, is this right? Seems natural that a GDM manager wants to access host's monitof config?

Comment (realmazharhussain) [reply io.github.realmazharhussain.GdmSettings.yaml:13]: Flathub doesn't allow that anymore.

Comment (bbhtt) [reply io.github.realmazharhussain.GdmSettings.yaml:13]: Yes please ask for an exception, the exception system is in place for exactly this.

Comment (realmazharhussain) [reply io.github.realmazharhussain.GdmSettings.yaml:13]: I did not know there was such a system before #37. Now, I have asked for an exception. https://github.com/flathub-infra/flatpak-builder-lint/pull/317

**Pull request merged**